homeassistant/helpers/template.py

```diff
@@ -423,6 +424,17 @@ def multiply(value, amount):
         return value
 
 
+def logarithm(value, base=math.e):
+    """Filter to get logarithm of the value with a spesific base."""
+    try:
+        if base == 10:
+            return math.log10(float(value))
+        else:
```

Comment:
I think you can do `return math.log(float(value), float(base)) `for base==10 also.

Reply:
Hi @Danielhiversen,

Fair point. It will result in cleaner code.. I used `log10` because it is more [accurate](https://docs.python.org/3/library/math.html#math.log10) than using `log(value, 10)`, but it more or less doesn't matter.

```python
>>> math.log10(float(1000))            
3.0                                    
>>> math.log(float(1000),10)
2.9999999999999996
```

I can change the implementation, and fix the test (one test will fail because of the behavior described). It is no problem, since all use of this will round the values in one way or another.